visidata/loaders/postgres.py

```diff
@@ -84,18 +88,61 @@ def postgresGetColumns(vd, cur):
         yield ColumnItem(coldesc.name, i, type=codeToType(coldesc.type_code, coldesc.name))
 
 
+class PgSchemataSheet(Sheet):
+    rowtype = 'schemas'
+
+    def reload(self):
+        if options.postgres_schema == "*":
```

Comment:
Use `self.options` (here at least, also other places it's used) so that the option can be overridden on a per-sheet basis.

Reply:
And let's make it so that an empty string means all-but-internal schema, instead of `*`.  Maybe `*` can mean "including internal schemas" like you have for `**` below.  Do we think it's important to support internal schemas though?